torch_np/tests/numpy_tests/core/test_scalarmath.py

```diff
@@ -352,18 +357,10 @@ def test_float_modulus_corner_cases(self):
                     div, mod = op(fone, fzer)
                     assert_(np.isinf(div)) and assert_(np.isnan(mod))
 
-    def test_inplace_floordiv_handling(self):
-        # issue gh-12927
-        # this only applies to in-place floordiv //=, because the output type
-        # promotes to float which does not fit
-        a = np.array([1, 2], np.int64)
-        b = np.array([1, 2], np.uint64)
-        with pytest.raises(TypeError,
-                match=r"Cannot cast ufunc 'floor_divide' output from"):
-            a //= b
-
 
 class TestComplexDivision:
+
+    @pytest.mark.xfail(reason='With pytorch, 1/(0+0j) is nan + nan*j, not inf + nan*j')
```

Comment:
probably simply skip. These are the sort of incongruences we should be alright with having.